encoders/nlp/TransformerTorchEncoder/requirements.txt

```diff
@@ -1,2 +1 @@
-transformers==4.1.1
-torch==1.7.1
+jina
```

Comment:
why can't the dependencies be kept here?

Reply:
This way both CUDA and torch can be installed together very easily. This way of installing is also the recommended way listed on pytorch website.